packages/database/src/migrations/20210927003701-LESMIS5YearOldEnrolmentRateMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,201 @@
+          value: '=divide($value, 100)',
+        },
+        exclude: ['organisationUnit', 'dataElement', 'period'],
+      },
+    ],
+  },
+});
+
+const getMapOverlay = (name, reportCode) => ({
+  id: reportCode,
+  name,
+  userGroup: PERMISSION_GROUP,
+  dataElementCode: 'value',
+  isDataRegional: true,
+  legacy: false,
+  report_code: reportCode,
+  measureBuilder: 'useReportServer',
+  measureBuilderConfig: {
+    dataSourceType: 'custom',
+    reportCode,
+  },
+  presentationOptions: {
+    scaleType: 'performance',
+    displayType: 'shaded-spectrum',
+    measureLevel: 'SubDistrict',
+    valueType: 'percentage',
+    scaleBounds: {
+      left: {
+        max: 'auto',
+      },
+      right: {
+        min: 'auto',
+      },
+    },
+    periodGranularity: 'one_year_at_a_time',
+  },
+  countryCodes: '{"LA"}',
+  projectCodes: '{laos_schools}',
+
+  legacy: false,
+  report_code: reportCode,
+});
+
+const addMapOverlayGroup = async (db, parentCode, { name, code }) => {
+  const parentId = await codeToId(db, 'map_overlay_group', parentCode);
+
+  const overlayGroupId = generateId();
+  await insertObject(db, 'map_overlay_group', { id: overlayGroupId, name, code });
+
+  return insertObject(db, 'map_overlay_group_relation', {
+    id: generateId(),
+    map_overlay_group_id: parentId,
+    child_id: overlayGroupId,
+    child_type: 'mapOverlayGroup',
+  });
+};
+
+const addMapOverlay = async (db, parentCode, config) => {
+  const { reportCode, dataElements, name, sortOrder } = config;
+  const report = getReport(reportCode, dataElements);
+  const mapOverlay = getMapOverlay(name, reportCode);
+  const permissionGroupId = await nameToId(db, 'permission_group', PERMISSION_GROUP);
+  await insertObject(db, 'report', {
+    id: generateId(),
+    permission_group_id: permissionGroupId,
+    ...report,
+  });
+  await insertObject(db, 'mapOverlay', mapOverlay);
+
+  const mapOverlayGroupId = await codeToId(db, 'map_overlay_group', parentCode);
+  return insertObject(db, 'map_overlay_group_relation', {
+    id: generateId(),
+    map_overlay_group_id: mapOverlayGroupId,
+    child_id: mapOverlay.id,
+    child_type: 'mapOverlay',
+    sort_order: sortOrder,
+  });
+};
+
+exports.up = async function (db) {
+  // Add Map Overlays
+  for (const { parentCode, children } of MAP_OVERLAYS) {
+    for (const config of children) {
+      await addMapOverlay(db, parentCode, config);
+    }
+  }
+};
+
+const removeMapOverlay = (db, reportCode) => {
+  return db.runSql(`
+    DELETE FROM "report" WHERE code = '${reportCode}';
+    DELETE FROM "mapOverlay" WHERE "id" = '${reportCode}';
+    DELETE FROM "map_overlay_group_relation" WHERE "child_id" = '${reportCode}';
+  `);
+};
+
+const removeMapOverlayGroupRelation = async (db, groupCode) => {
+  const overlayId = await codeToId(db, 'map_overlay_group', groupCode);
+  await db.runSql(`
+    DELETE FROM "map_overlay_group_relation" WHERE "child_id" = '${overlayId}';
+  `);
+};
```

Comment:
Maybe get rid of these helper functions alongside deleting removing the `OVERLAY_GROUPS`